dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -578,18 +578,25 @@ private void updateInfo(Workflow oldWorkflow, Workflow newWorkflow) {
         @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = WorkflowVersion.class, responseContainer = "List")
     public Set<WorkflowVersion> requestDOIForWorkflowVersion(@ApiParam(hidden = true) @Auth User user,
         @ApiParam(value = "Workflow to modify.", required = true) @PathParam("workflowId") Long workflowId,
-        @ApiParam(value = "workflowVersionId", required = true) @PathParam("workflowVersionId") Long workflowVersionId) {
+        @ApiParam(value = "workflowVersionId", required = true) @PathParam("workflowVersionId") Long workflowVersionId,
+        @ApiParam(value = "This is here to appease Swagger. It requires PUT methods to have a body, even if it is empty. Please leave it empty.") String emptyBody) {
         Workflow workflow = workflowDAO.findById(workflowId);
         checkEntry(workflow);
         checkUser(user, workflow);
 
         WorkflowVersion workflowVersion = workflowVersionDAO.findById(workflowVersionId);
         if (workflowVersion == null) {
-            LOG.error(user.getUsername() + ": could not find version: " + workflow.getPath());
+            LOG.error(user.getUsername() + ": could not find version: " + workflow.getWorkflowPath());
```

Comment:
I tried to do this which is why it shows as a change, I got an error because its a null version and hence wouldn't have  a 'name' value. But this might be something for the UI to handle because what we actually in this request is the workflowVersionID (the number in the database) which isn't very helpful and would probably be more confusing to the user.

Reply:
I would say `"could not find version: " + workflowVersionId`. I think that is an appropriate message for the API -- you give the API an id, the API can't find the ID, and the error says what it can't find. The UI can decide what to display. 

If that's not appealing, then the message should not say `could not find version:` -- if I see the word `version` followed by a colon, the next thing I'm expecting to see is some version/version identifier.

Also, next line should probably be a 404, not a 400.

But this has gotten beyond the scope of this PR, as it was like this before, so you don't need to do any of this.

This was all like this before